include/fiction/technology/charge_distribution_surface.hpp

```diff
@@ -1158,19 +1167,19 @@ class charge_distribution_surface<Lyt, false> : public Lyt
      * @param history_mode charge_distribution_history::NEGLECT if the information (local electrostatic energy) of the
      * previous charge distribution is used to make the update more efficient, charge_distribution_history::CONSIDER
      * otherwise.
-     * @param engine exhaustive_sidb_simulation_engine::EXGS if *ExGS* should be used,
-     * exhaustive_sidb_simulation_engine::QUICKEXACT for *QuickExact*.
+     * @param engine exact_sidb_simulation_engine::EXGS if *ExGS* should be used,
+     * exact_sidb_simulation_engine::QUICKEXACT for *QuickExact*.
```

Comment:
This comment will run out of sync as soon as there is a new engine added or one removed (for whatever reason). Rather not mention the individual possibilities

Reply:
Holds similar for the other docstring, but not as strongly I would say. That is because it's not as likely that those `enum`s are updated.